runtime/sema/check_composite_declaration.go

```diff
@@ -196,72 +193,48 @@ func (checker *Checker) visitCompositeLikeDeclaration(declaration ast.CompositeL
 	checker.typeActivations.Enter()
 	defer checker.typeActivations.Leave(declaration.EndPosition)
 
-	if kind == ContainerKindComposite {
-		checker.enterValueScope()
-		defer checker.leaveValueScope(declaration.EndPosition, false)
-	}
+	checker.enterValueScope()
+	defer checker.leaveValueScope(declaration.EndPosition, false)
 
-	checker.declareCompositeLikeNestedTypes(declaration, kind, true)
+	checker.declareCompositeLikeNestedTypes(declaration, ContainerKindComposite, true)
```

Comment:
The container kind parameter for `declareCompositeLikeNestedTypes` seems to always be `ContainerKindComposite` now. I tried removing it in https://github.com/onflow/cadence/commit/f8e84d85d89998accbe31412ed41408a1bdeff01 and tests are still passing. Feel free to cherry pick it in, or we can open the clean up as a follow-up PR (if it is correct).